_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -528,6 +528,17 @@ def fips_to_megacounty(
         data = data.reset_index().groupby([date_col, mega_col]).sum()
         return data.reset_index()
 
+    def as_mapper_name(self, geo_type):
```

Comment:
Are we sure "state" -> "state_code" is always the intended mapping?

Reply:
hmmm it should probably be state_id by default; i always get that wrong.

Added an override. For this use case it doesn't matter, but for others it might.

Reply:
got it. what do you think of making as_mapper_name internal to geomapper so the mapping functions natively accept "state" and "county" (would address #628), and geomapper users dont need to worry about using it? Something like calling it as the first line to the add/replace/get geocode values. I think the argument against is that it's a bit less explicit and you cant easily choose something like  'state' -> 'state_name'. Don't feel that strongly so will go ahead and approve this for now.